website/templates/project/contributors.mako

```diff
@@ -353,8 +356,10 @@
 
                 <!-- ko ifnot: contributor.canEdit() -->
                     <!-- ko if: canRemove -->
```

Comment:
All of them are Knockout [computed observables](http://knockoutjs.com/documentation/computedObservables.html).